### ISSUE TITLE
Image normalisation while predicting in demo.py

### ISSUE THREAD

**Issue by ankushpanwar19** [kirillProkofiev/light-weight-face-anti-spoofing#30]
I checked the demo.py script and its related files but I don't see any image normalization while prediction. However in train.py I could see image normalization using albumentations.
Is there a specific reason why normalization has been removed from the inference?

**Comment by stakanich** [comment]
Hi. Maybe too late but I have an answer for you.
In demo.py you can see the definition of the spoof_model:
```
    if args.spf_model.endswith('pth.tar'):
        if not args.config:
            raise ValueError('You should pass config file to work with a Pytorch model')
        config = utils.read_py_config(args.config)
        spoof_model = utils.build_model(config, args, strict=True, mode='eval')
        spoof_model = TorchCNN(spoof_model, args.spf_model, config, device=device)
    else:
        assert args.spf_model.endswith('.xml')
        spoof_model = VectorCNN(args.spf_model)
```
and in demo_tools/wrapers.py
```
class TorchCNN:
    ...
    def preprocessing(self, images):
        ''' making image preprocessing for pytorch pipeline '''
        mean = np.array(object=self.config.img_norm_cfg.mean).reshape((3,1,1))
        std = np.array(object=self.config.img_norm_cfg.std).reshape((3,1,1))
        height, width = list(self.config.resize.values())
        preprocessed_imges = []
        for img in images:
            img = cv.resize(img, (height, width) , interpolation=cv.INTER_CUBIC)
            img = cv.cvtColor(img, cv.COLOR_BGR2RGB)
            img = np.transpose(img, (2, 0, 1)).astype(np.float32)
            img = img/255
            img = (img - mean)/std
            preprocessed_imges.append(img)
        return torch.tensor(preprocessed_imges, dtype=torch.float32)

    def forward(self, batch):
        batch = self.preprocessing(batch)
        self.model.eval()
        model1 = (self.model.module
                  if self.config.data_parallel.use_parallel
                  else self.model)
        with torch.no_grad():
            output = model1.forward_to_onnx(batch)
            return output.detach().numpy()
```

So, at least for .pth.tar models, here you can see normalization.
